### PR TITLE
os/wqueue: fix typo

### PR DESCRIPTION
**Pull request by dev-hue**
- fix typo of comment

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [c31ad561f788022257a2f1fe204147d84a42fd05] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [c31ad561f788022257a2f1fe204147d84a42fd05] - Code Rule Check (C++) OK.

File: os/wqueue/work_queue.c [hunk 1]
```diff
@@ -92,7 +92,7 @@
  *
  * Description:
  *   Queue work to be performed at a later time.  All queued work will be
- *   performed on the worker thread of of execution (not the caller's).
+ *   performed on the worker thread of execution (not the caller's).
```
Comment (jsdosa) [inline os/wqueue/work_queue.c]: looks good to me.

**Comment by YiHyunJin** [comment]
Good

File: os/wqueue/work_queue.c [hunk 2]
```diff
@@ -92,7 +92,7 @@
...
  *
```
Comment (hasw7569) [inline os/wqueue/work_queue.c]: LGTM

**Pull request merged**